presto-main/src/main/java/com/facebook/presto/metadata/OperatorSignatureUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.metadata;
+
+import com.facebook.presto.spi.function.OperatorType;
+import com.google.common.annotations.VisibleForTesting;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+public final class OperatorSignatureUtils
```

Comment:
Method references will look nicer if we name this `OperatorSignatures`

Reply:
We normally do static import of static methods so the class name would not appear at the site of method reference. I don't have strong opinions of this class name though. I have a strong feeling that this class would go away some day anyways. I can rename if it would make you happier.